### DISCUSSION TITLE
CC2538 when debugging, using what IDE software debugging, anyone know?

### DISCUSSION THREAD

**Discussion by slcmcu** [contiki-ng/contiki-ng#2395] [General]
CC2538 when debugging, using what IDE software debugging, anyone know?

**Comment by slcmcu** [comment]
What method to use  contiki cc2538 debug ?

**Comment by rajeev1986** [comment]
I think once you are able to do this:
https://docs.contiki-ng.org/en/develop/doc/platforms/simplelink.html?highlight=code%20composer#set-up-contiki-ng-in-code-composer-studio

then you should be able to use debug feature of CCS.
I have not tried this with Contiki, but have used CCS debug feature.
